lessons/codap-transformers2/langs/en-us/pages/codap-transform.adoc

```diff
@@ -0,0 +1,83 @@
+= Transform Attribute
+++++
+<style>
+body.workbookpage td .autonum::after { content: ')' !important; }
+/* Cram everything to the top instead of distributing space equally */
+#content { display: block !important; }
+</style>
+++++
+Make sure you’re logged into the @starter-file{animals} in CODAP. Select the Plugins icon, then choose Transformers.
+
+@vspace{1ex}
+
+== Create, Apply, and Save a Transform Attribute Transformer (Step by Step)
+
+[cols="5, 10", frame="none", grid="none", stripes="none"]
+|===
+
+|
+@image{../images/transform-ui-annotated.png, 350}
+
+|
+@n Choose Transform Attribute from the drop-down menu.
+
+@vspace{1ex}
+
+@n We want to create a Transformer that will replace all ages less than 5 with the Boolean `true`. In other words, it will _transform_ our "age" column into a column that tells us if an animal is young or not. What is a good name for this Transformer?
+
+@fitb{}{@ifsoln{transform-is-young}}
+
+@vspace{1ex}
+
+@n Select the dataset you'd like to transform.
+
+
+@n What attribute will we be transforming? @fitb{}{@ifsoln{Age}}
+
+@vspace{1ex}
+
+@star Select the attribute, paying attention the sentence we've starred (left). What happened?
+
+@fitb{}{}
+
+@n What would be an appropriate name for our transformed attribute? @fitb{}{@ifsoln{`Young`}}
+
+@n The contract includes a Domain (row) only. What is the Range? @fitb{}{@ifsoln{`Boolean`}
+
+@vspace{1ex}
+
+@n Let's write a purpose statement: _The expression consumes @fitb{10em}{@ifsoln{a row}}, and produces @fitb{10em}{@ifsoln{true}} if the age is less than 5._
+
+@n What is the expression? @fitb{}{@ifsoln{`Age < 5`}}
+
+@vspace{1ex}
+
+@n Apply the Transformer.
+
```

Comment:
I'm noticing that the option exists to `save the Transformer` and I wonder when you would or wouldn't encourage students to save.